src/miner.cpp

```diff
@@ -142,7 +142,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
       CBlockHeader h;
       nBlockSize += h.GetSerializeSize(SER_NETWORK, PROTOCOL_VERSION);
     }
-    assert(nBlockSize == 80);  // BU always 80 bytes
+    assert(nBlockSize == 80);  // BU block header is always 80 bytes
```

Comment:
Is this really a consistent argument, seeing as MAX_COINBASE_SCRIPTSIG_SIZE has been applied elsewhere in place of its magic number? I think good practice should take precedence over complaints here.